packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.module.js

```diff
@@ -1,5 +1,5 @@
 import angular from 'angular';
-
+import 'ovh-api-services';
```

Comment:
**nitpick:** `ovh-api-services` is deprecated. Using `$http` instead.

Reply:
Ah yes I removed it from package.json but forget to remove the imports